src/services/file.forked.services.js

```diff
@@ -19,14 +24,25 @@ import {
  */
 const getFileFromForkedRepoByPath = async (owner, repo, branch, path) => {
   try {
-    const { data } = await axios.get(`${GITHUB_APP_BASE_URL}/files`, {
-      params: {
-        owner: owner,
-        repo: repo,
-        branch: branch,
-        path: path,
-      },
-    });
+    if(!GITHUB_APP_BASE_URL || GITHUB_APP_BASE_URL.trim() === '') {
+      throw new MissingGitHubAppDomainError();
+    }
+    if(!CHANGELOG_PR_BRIDGE_SECRET_KEY || CHANGELOG_PR_BRIDGE_SECRET_KEY.trim() === '') {
+      throw new MissingChangelogBridgeSecretKeyError();
+    }
+    const { data } = await axios.get(
```

Comment:
Not related to this PR but why did you go with `axios` here? node has a default `fetch` api that you can use.

Reply:
I prefer `axios`, honestly. Later, we plan to introduce an Axios client to reduce code duplication. Any concern with this selection? We can refactor of course, opening an issue.

Reply:
Im okay either way. I try to usually have as few dependencies as possible since maintaining them is a pain. So unless i have a really good reason for needing a dependency, i try to avoid it in general. And with fetch being natively suppored in NodeJS now, I dont really see a need for axios usually. This was not the case before NodeJS supported the fetch api.

Reply:
You have a point. We can open an issue to replace that. Just use `axios` because I like its feature for that has for instances and interceptors, so we can have a neater code. But if you want to go with `fetch` we change to that. We follow your lead.